CMakeLists.txt

```diff
@@ -106,6 +107,9 @@ if( EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
   # conanfile.txt
   #
   conan_basic_setup( TARGETS KEEP_RPATHS )
+
+  # REVISIT (sparent) : This should not be necessary but appears to be
+  set(CMAKE_CXX_STANDARD ${CONAN_SETTINGS_COMPILER_CPPSTD})
```

Comment:
This should not be necessary but I cannot get the Conan CMake generator to propagate the `CMAKE_CXX_STANDARD` flag per the docs here: https://docs.conan.io/en/latest/integrations/build_system/cmake.html